openslides/users/static/templates/users/user-list.html

```diff
@@ -50,37 +30,28 @@ <h1 translate>Participants</h1>
       </button>
     </div>
     <div class="col-sm-6">
-      <div class="form-inline text-right">
-        <div class="form-group">
-          <div class="input-group">
-            <div class="input-group-addon"><i class="fa fa-search"></i></div>
-            <input type="text" ng-model="filter.search" ng-model-options="{debounce: 500}" class="form-control"
-               placeholder="{{ 'Search' | translate}}">
-          </div>
-        </div>
-        <button class="btn btn-default" ng-click="$parent.isFilterOpen = !$parent.isFilterOpen"
-            ng-class="$parent.isFilterOpen ? 'btn-primary' : 'btn-default'">
-          <i class="fa fa-filter"></i>
-          <translate>Filter ...</translate>
+      <div class="dropdown pull-right" uib-dropdown>
+        <button os-perms="users.can_manage" class="btn btn-default" uib-dropdown-toggle
+            id="user-pdf-dropdown">
+          <i class="fa fa-file-pdf-o fa-lg"></i>
+          <translate>PDF Export</translate>
```

Comment:
And maybe you can easily add a nice csv export of filtered data - like in motions?